enarx-keep-sgx-shim/src/handler.rs

```diff
@@ -321,12 +321,13 @@ impl<'a> Handler<'a> {
             }
         }
 
-        let utsname = unsafe { &mut *(self.aex.gpr.rdi.raw() as *mut libc::utsname) };
-        fill(&mut utsname.sysname, "Linux");
-        fill(&mut utsname.nodename, "localhost.localdomain");
-        fill(&mut utsname.release, "5.6.0");
-        fill(&mut utsname.version, "#1");
-        fill(&mut utsname.machine, "x86_64");
+        let u: *mut libc::utsname = self.aex.gpr.rdi.into();
+        let u = unsafe { &mut *u };
```

Comment:
This is interesting.

Reply:
What's interesting? We have `Into` conversions for pointers from registers. But registers can't tell us lifetimes. Therefore, it is unsafe to convert a pointer to a reference because this requires a pointer dereference.

Reply:
Thanks!